### DISCUSSION TITLE
ImportError ProcgenEnv

### DISCUSSION THREAD

**Discussion by nforsg** [vwxyzjn/cleanrl#447] [General]
Hey, 

I am working with this awesome library, but have encountered a wierd error that I have been unable to solve when running the ppg_procgen.py file. 

I get the error: "ImportError: cannot import name 'ProcgenEnv' from 'procgen'", when running the script, which means that probably there is some kind of a bug in the source code for procgen benchmark right? I.e the have managed to make like a circular import entangelement or similar. Have anyone managed to run the code without this error? I would be very thankful if you could give me any hints of what this problem might depend on. The installation goes completely fine on python 3.12.1.

**Comment by pseudo-rnd-thoughts** [comment]
Have you downloaded procgen? Just run `from procgen import ProcgenEnv` in a python terminal

**Reply by nforsg** [reply]
Yes I have. I am using Python 3.12

**Reply by pseudo-rnd-thoughts** [reply]
As procgen is completely unrelated to cleanrl, then there shouldn't be any issues
